src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+public class Deadline extends Task {
+
+    protected String by;

```

Comment:
Naming - Consider using a more intuitive name such as "dueDate" instead of "by", since "by" used in certain context may not have a date meaning attached to it